src/main/java/seedu/address/logic/commands/ResetCommand.java

```diff
@@ -22,44 +24,79 @@ public class ResetCommand extends Command {
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Resets attendance for the contact "
             + "by the index number used in the last person listing and for the tutorial number inputted. "
-            + "Parameters: INDEX (must be a positive integer) "
-            + "tut/TUTORIAL\n"
+            + "Parameters: \n"
+            + "INDEX (must be a positive integer) \n"
+            + "tut/TUTORIAL in the format of \n"
+            + "1) A positive number between 1-12 \n"
+            + "2) A list of numbers eg. [1,3,5] \n"
+            + "3) A range of two numbers eg. 3-6 \n"
             + "Example: " + COMMAND_WORD + " 1 "
-            + "tut/1";
+            + "tut/1-5";
 
     private final Index index;
-    private final Tutorial tutorial;
+    private final List<Tutorial> tutorials;
 
     /**
      * @param index The index of the person to be marked.
-     * @param tutorial The tutorial to set as not taken place.
+     * @param tutorial single tutorial to set as not taken place.
      */
     public ResetCommand(Index index, Tutorial tutorial) {
         this.index = index;
-        this.tutorial = tutorial;
+        this.tutorials = List.of(tutorial);
+    }
+
+    /**
+     * Constructor for resetting multiple tutorials.
+     * @param index of the person in the display list.
+     * @param tutorials list of tutorials to set as not taken place.
+     */
+    public ResetCommand(Index index, List<Tutorial> tutorials) {
+        this.index = index;
+        this.tutorials = tutorials.stream().distinct().collect(Collectors.toList());
     }
 
     /**
      * Takes a person and tutorial and resets the specified tutorial and returns that person with the new tutorials.
      * @param personToEdit The person whose attendance will be reset.
-     * @param tutorial The tutorial to reset attendance for.
+     * @param tutorials tutorials to reset attendance for.
      */
-    private Person generateResetPerson(Person personToEdit, Tutorial tutorial) throws CommandException {
+    private Pair<Person, String> generateResetPerson(Person personToEdit,
```

Comment:
Same comments as above, and since we have duplicated logic, perhaps we can put this logic into a util class method so both command classes can use it with their respective arguments.